README.md

```diff
@@ -227,15 +227,16 @@ $ interpreter
 
 In the interactive mode, you can use the below commands to enhance your experience. Here's a list of available commands:
 
- **Available Commands:**                                                         
- • `%debug [true/false]`: Toggle debug mode. Without arguments or with 'true', it 
-   enters debug mode. With 'false', it exits debug mode.                        
- • `%reset`: Resets the current session.                                          
- • `%save_message [path]`: Saves messages to a specified JSON path. If no path is 
-   provided, it defaults to 'messages.json'.                                    
+**Available Commands:**  
+ • `%debug [true/false]`: Toggle debug mode. Without arguments or with 'true', it
+enters debug mode. With 'false', it exits debug mode.  
+ • `%reset`: Resets the current session.  
+ • `%save_message [path]`: Saves messages to a specified JSON path. If no path is
+provided, it defaults to 'messages.json'.  
  • `%load_message [path]`: Loads messages from a specified JSON path. If no path  
-   is provided, it defaults to 'messages.json'.                                 
+ is provided, it defaults to 'messages.json'.  
  • `%help`: Show the help message.
+• `%undo`: Remove previous messages and its response from the message history.
```

Comment:
Maybe relocate it to below %reset, as they are the same type of command, and leave the help as the last item